comps/llms/text-generation/predictionguard/README.md

```diff
@@ -0,0 +1,54 @@
+# Introduction
```

Comment:
shall we enable this by default but not introduce a new instance under llms/text-generation?